posix.mak

```diff
@@ -79,7 +79,9 @@ MOD_EXCLUDES_PRERELEASE=$(addprefix --ex=, gc. rt. core.internal. core.stdc.conf
 	std.algorithm.internal std.c. std.concurrencybase std.internal. std.regex.internal.  \
 	std.windows.iunknown std.windows.registry etc.linux.memoryerror	\
 	std.experimental.ndslice.internal std.stdiobase \
-	tk. msvc_dmc msvc_lib)
+	tk. msvc_dmc msvc_lib \
+	ddmd.libmach ddmd.libmscoff ddmd.objc_glue \
+	ddmd.scanmach ddmd.scanmscoff)
```

Comment:
I had a quick look at whether it was possible to enable them. At least in the case of the OS X stuff, it was importing stuff from Druntime, and those modules had a `version(OSX):` on top.

I'm not sure if that makes sense for things such as type declarations; I have a Druntime PR open to make the Windows headers usable on other platforms (for things such as .bmp and .exe/.dll file formats), but that grew the size of the built library file by a disproportionate amount due to all the `.init` / `TypeInfo` clutter...